src/extensions/google-shopping-ads/index.md

```diff
@@ -3,6 +3,9 @@ group: extensions
 title: Install Google Shopping ads Channel
 ---
 
+{:.bs-callout .bs-callout-info}
```

Comment:
```suggestion
{:.bs-callout-warning}
```

Reply:
agreed to stay blue to be consistent with merchdocs banner

Reply:
not sure what consistency do you mean
it's warning on merchdocs:
https://github.com/magento/merchdocs/blob/master/src/sales-channels/google-ads/google-ad-channel.md

Reply:
the header bar that has this same text is in a blue bar at the top of every page of the Google section. and when shown to the product manager in the orange, he asked why it was different, so i changed it back to blue.